lib/ansible/plugins/callback/default.py

```diff
@@ -47,7 +48,20 @@ def __init__(self):
         super(CallbackModule, self).__init__()
 
     def v2_runner_on_failed(self, result, ignore_errors=False):
```

Comment:
instead doing this on the default callback, i would update the base class

Reply:
Sounds good.